### PR TITLE
Minor install instruction fixes

### PR DESCRIPTION
**Pull request by jsancio**
## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [x] High
- [ ] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping @emanic @sascala @joel-hamill for reviewing pull request changes.

### CODE REVIEW

**Comment by joel-hamill** [comment]
@jgarcia-mesosphere the convention in our docs (currently) is to prefix commands with the `$`. This is pretty common in technical doc (e.g. https://docs.docker.com/compose/gettingstarted/).

**Comment by lingmann** [comment]
Other than comment around $releasever this LGTM

**Comment by mhausenblas** [comment]
@joel-hamill did you maybe mean @jsancio?

**Comment by jsancio** [comment]
@joel-hamill Done. I still don't think we should add `$` but okay. I can think of two reason why removing $ is nice. 1) easy to copy and paste. 2) the user doesn't type `$`, it is printed by their terminal.

**Comment by jsancio** [comment]
@lingmann Did you mean ".... `$`"? What comment around `$releasever`?

**Comment by lingmann** [comment]
sorry @jsancio my comment was stuck in a pending review... see above.

**Comment by lingmann** [comment]
Also FWIW I agree with Jose that our convention of `$` prefix for commands is not ideal for the reasons he stated. Would love to re-evaluate this convention at some point.

File: 1.8/administration/installing/custom/system-requirements/install-docker-centos.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following instructions demonstrate how to use Docker with OverlayFS on CentO
     $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
     [dockerrepo]
     name=Docker Repository
-    baseurl=https://yum.dockerproject.org/repo/main/centos/$releasever/
+    baseurl=https://yum.dockerproject.org/repo/main/centos/7/
```
Comment (jsancio) [inline 1.8/administration/installing/custom/system-requirements/install-docker-centos.md]: I tried that and it got replaced with `7server` or something like that which resulted in a broken URL. The instructions are very CentOS7 specific anyways so I didn't see a problem with this fix.

Comment (lingmann) [inline 1.8/administration/installing/custom/system-requirements/install-docker-centos.md]: Ok sounds good. Using just `7` matches what Docker currently recommends here: https://docs.docker.com/engine/installation/linux/centos/

**Pull request merged**